src/plasma/status.h

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
```

Comment:
Right, but the copyright seems inaccurate regardless of which codebase it is in, right?